vue/src/store.ts

```diff
@@ -182,7 +182,9 @@ export const state = reactive<{
   modelRuns: KeyedModelRun[],
   totalNumModelRuns: number;
   openedModelRuns: Set<KeyedModelRun["key"]>;
-  gifSettings: { fps: number, quality: number, pointSize: number},
+  gifSettings: { 
+    fps: number,
+    pointSize: number,},
```

Comment:
Formatting nit here. Probably need to fix the js linter and checker at some point.